### ISSUE TITLE
Mapping velden rechtspraak

### ISSUE THREAD

**Issue by mvwestendorp** [mvwestendorp/v-en-a#26]
Velden voor rechtspraak zijn nu logischer in het Engels dan in het Nederlands. 
Herbekijken in functie van Nederlandse taalinstelling.

Zie ook #17

**Comment by mvwestendorp** [comment]
Voorstel mapping (buiten de evidente waarden):
commentator: auteur van de noot.
auteur: auteur van de concl.
jurisprudentietijdschrift: tijdschriftnaam
jaargang (, editie): voor jaar en editie tijdschrift

Jaargang indien dat de invulling in dat vak dat toelaat van zowel het jaartal als de editie op een duidelijk manier.

**Comment by mvwestendorp** [comment]
Dit zouden de nieuwe vakken zijn. 
![image](https://user-images.githubusercontent.com/902633/55220043-9edbd800-5206-11e9-817f-76bee7f5d35c.png)

**Comment by ElisabethdL** [comment]
Waarom niet beginpagina bij "eerste pagina"? 
Editie zou vrij moeten zijn voor de aflevering, indien nodig.
Verder prima, denk ik.

**Comment by mvwestendorp** [comment]
Ja, dat moet eigenlijk ook. Ik was nog met afleveringen aan het proberen. Zo zou het dan worden:
![image](https://user-images.githubusercontent.com/902633/55233523-32260500-5229-11e9-90a8-e4c7b45aab23.png)

**Comment by ElisabethdL** [comment]
De "afl." wordt er vanzelf bijgezet, denk ik? Dus moet je niet zelf invullen.

**Comment by mvwestendorp** [comment]
Nee, in dit geval niet. Daar is ook niet direct iets aan te doen denk ik.

**Comment by ElisabethdL** [comment]
Gaat wel bij tijdschriftartikels, zie ik.

**Comment by mvwestendorp** [comment]
Dat komt hierdoor:
> <number variable="issue" prefix=", afl. "/>

In het geval van rechtspraak wordt dat niet altijd vermeld en gaat het soms over de pagina en soms over de aflevering. Maar wat precies bedoeld wordt  is niet geweten als er alleen een getal wordt ingevoerd bij de huidige wijze van invullen. In de waardes zelf kan ik niets tussenvoegen.

**Comment by mvwestendorp** [comment]
Verbeteringen in mapping hier: 30225ffefde95fa7db66fb2ec9d0c4d11fb53c16
Documentatie aanpassing volgt.

**Comment by ElisabethdL** [comment]
Auteur van noot als "commentator": na het woord "noot" staat een punt teveel.

> Cass. 27 januari 2011, nr. AR F.07.0109.F, Belgische Staat/Fortis, noot. R. HOUBEN.

**Comment by ElisabethdL** [comment]
Hoe verwijzen naar een anonieme noot?
Bijvoorbeeld V&A 109 (laatste) & 110
> Arbrb. Nijvel 4 oktober 1994, _Rev.dr.étr._ 1994, 383, noot.
> Antwerpen (8e k.) 9 september 1993, _RW_ 1993-94, 406, noot.
